appcues-react-native.podspec

```diff
@@ -16,6 +16,5 @@ Pod::Spec.new do |s|
   s.source_files = "ios/**/*.{h,m,mm,swift}"
 
   s.dependency "React-Core"
-  # NOTE: remove the specific version requirement once 1.0.0 is published
-  s.dependency "Appcues", "1.0.0-beta.4"
+  s.dependency "Appcues"
```

Comment:
nice on line 7 above how it just pulls in from the package version... couldn't see how to do this with Flutter yet

Reply:
Wasn't my work (it's part of the template), but I'll agree that it's nice 👍